api/api.proto

```diff
@@ -100,6 +100,26 @@ message VersionResp {
   int32 api = 2;
 }
 
+// RefreshTokenRef contains the metadata for a refresh token that is managed by the storage.
+message RefreshTokenRef {
+  // ID of the refresh token.
+  string id = 1;
+  string client_id = 2;
+  string created_at = 3;
+  string last_used = 4;
+}
+
+// ListRefreshReq is a request to enumerate the refresh tokens of a user.
+message ListRefreshReq {
+  string user_id = 1;
```

Comment:
nit: need to update this struct to only take a user_id. Maybe a comment like this?

```
message ListRefreshReq {
    // The "sub" claim returned in the ID Token
    string user_id = 1;
}
```